tinder.js

```diff
@@ -62,10 +65,29 @@ function TinderClient() {
    * @param {String} path the relative path
    * @param {Object} data an object containing extra values
    * @param {Function} callback the callback to invoke when the request completes
+   * @param {Object} form the form to send if we need multipart format
    */
-  var tinderPost = function(path, data, callback) {
-    var opts = getRequestOptions(path, data);
+  var tinderPost = function(path, data, callback, form) {
```

Comment:
For sure but since, we can't put a body in multipart request it will be more like that : 
```
var tinderMultipartPost = function(path, formData, callback) {
    var opts = getRequestOptions(path, null);
    opts['url'] = TINDER_IMAGE_HOST + path,
    opts['formData'] = formData;
    opts.method = 'POST';
    request(opts, callback);
  };
```